assets/js/src/newsletter_editor/components/config.js

```diff
@@ -32,7 +32,7 @@ App.on('before:start', function (BeforeStartApp, options) { // eslint-disable-li
 
   config.blockDefaults = _.extend(
     config.blockDefaults,
-    options.newsletter.body.blockDefaults || {}
+    (options.newsletter.body ? options.newsletter.body.blockDefaults : null) || {}
```

Comment:
I think it would be nicer if we used the same fallback value for both cases. I guess we can go with an empty object.
We could also simplify it by using optional chaining syntax `options.newsletter.body?.blockDefaults || {}`

Reply:
I had a solution with optional chaining syntax at the begging, but eslint didn't allow it. I am going to try it again and I will investigate why.

Reply:
Oh! I read the condition incorrectly 🤦 . It is ok as is. My bad! Sorry. If you want to add the optional chaining please do so and switch back to me or just switch it back to me.